src/Rokt-Kit.js

```diff
@@ -305,4 +373,5 @@ if (window && window.mParticle && window.mParticle.addForwarder) {
 
 module.exports = {
     register: register,
+    OnboardingExpProvider: OnboardingExpProvider,
```

Comment:
is this necessary?  If it's only necessary for the tests, I think you can just duplicate the values in the test itself.

This is generally just used for an end developer to use.